### PR TITLE
Update the auth info screen

### PR DESCRIPTION
**Pull request by ziccardi**
As of issue [AEROGEAR-2972](https://issues.jboss.org/browse/AEROGEAR-2972):

- [x] Change the title of the page to 'Identity Profile'
- [x] apply the same as the Cordova screen in Slide 8 in [slideset](https://docs.google.com/presentation/d/1i5mRe2n0PvmhhEZn-_o14g9GVLDzeFiNn7AC8nL1PQQ/edit#slide=id.g3b829bc65e_1_8)
- [x] Add the OTP Enabled and Email Verified fields across all platforms (these currently exist in Cordova). These are based on user data returned in the JWT from Keycloak.
- [x] *Roles Screen Section:* based on the Cordova screen in Slide 8 in [slideset](https://docs.google.com/presentation/d/1i5mRe2n0PvmhhEZn-_o14g9GVLDzeFiNn7AC8nL1PQQ/edit#slide=id.g3b829bc65e_1_7), add a Header to that section titled 'Assigned Roles', across all platforms.
- [x] Update Slideset (both android and iOS)
- [x] Remove previous page title from user details page

### CODE REVIEW

**Comment by ziccardi** [comment]
@ciaranRoche Nice catch! The issue is that the 'Authentication' string is the name of the previous screen, and is visualised by iOS together with the `back arrow` (another difference I just noticed thanks to your comment)

**Comment by aidenkeating** [comment]
@ciaranRoche Looks like it is meant to be `Identity Profile`, I'll update the iOS app.

File: Example/Views/Pages/UserDetails.xaml [hunk 1]
```diff
@@ -1,34 +1,73 @@
...
                 </StackLayout>
             </StackLayout>
-            <circleimg:CircleImage x:Name="Image" Source="aerogear_icon.png" Aspect="AspectFill" WidthRequest="80" HeightRequest="80" HorizontalOptions="Center" RelativeLayout.XConstraint="{ConstraintExpression Type=RelativeToParent, Property=Width, Factor=0.5, Constant=-40}" RelativeLayout.YConstraint="{ConstraintExpression Type=RelativeToParent, Property=Y, Factor=0, Constant=16}" />
-        </RelativeLayout>
-    </ContentPage.Content>
+        </StackLayout>
+        <StackLayout Spacing="0" Margin="10,0,10,0" BackgroundColor="White">
+
+            <!--Email-->
+            <Label BackgroundColor="White" TextColor="{DynamicResource Primary}" FontSize="Micro" Text="Email" Margin="10,10,10,0" />
```
Comment (wojta) [inline Example/Views/Pages/UserDetails.xaml]: We can now use also StringResources from XAML. It's dirty solution and works only for Text attribute, but it's based on the Xamarin documentation (simplified by fact that there is no translation to different languages).

xmlns:s="clr-namespace:Example.Utils;assembly=Example"
```xml
<Label Text="{s:StringResource UnderConstructionTitle}" 
```

File: Example/Views/Pages/UserDetails.xaml.cs [hunk 2]
```diff
@@ -1,30 +1,57 @@
...
+    public partial class UserDetails : ContentPage, INotifyPropertyChanged
     {
+        private User user;
+        private ObservableCollection<UserRole> userRoles = new ObservableCollection<UserRole>();
+
         public UserDetails()
         {
+            this.user = MobileCore.Instance.GetService<IAuthService>().CurrentUser();
+            userRoles = new ObservableCollection<UserRole>(this.user.Roles);
+            //NotifyPropertyChanged(nameof(AssignedRoles));
```
Comment (wojta) [inline Example/Views/Pages/UserDetails.xaml.cs]: maybe remove before merge

File: Example/Views/Pages/UserDetails.xaml.cs [hunk 3]
```diff
@@ -1,30 +1,57 @@
...
             InitializeComponent();
-            InitListView();
+            BindingContext = this;
+            //NotifyPropertyChanged(nameof(AssignedRoles));
```
Comment (wojta) [inline Example/Views/Pages/UserDetails.xaml.cs]: here as well

File: Example/Views/Pages/UserDetails.xaml.cs [hunk 4]
```diff
@@ -1,30 +1,57 @@
...
+            //InitListView();
         }
 
-        private void InitListView() {
-            var user = MobileCore.Instance.GetService<IAuthService>().CurrentUser();
-            var roleItems = new List<string> { };
-            foreach(var role in user.getRoles())
+        //private void InitListView()
```
Comment (wojta) [inline Example/Views/Pages/UserDetails.xaml.cs]: and here

**Comment by ziccardi** [comment]
@wei-lee mind giving a look? I've updated the screenshots as well: https://docs.google.com/presentation/d/1FZL-Wy5D_wUo5ADrfhpRBh-Ny2rGauI3_C8SBrlk6rw/edit#slide=id.g3c52d834c7_0_65

**Pull request merged**